File: 第二章Swift介绍与UI设计/实验2：创建简单注册表单/MZRegistrationForm/MZRegistrationForm/ViewController.m
//
//  ViewController.m
//  MZRegistrationForm
//
//  Created by 祝海焜 on 15/12/8.
//  Copyright © 2015年 Matin.Zhou. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)TouchButton:(id)sender {
    // action on clickOnButton
    _EnterName.text = @"";
    _EnterEmail.text = @"";
    _EnterPhone.text = @"";
    _EnterCompany.text = @"";
    _EnterId.text = @"";
    _EnterAddress.text = @"";
}

- (IBAction)Submit:(id)sender {
    // Action when tap on background
    [self.view endEditing:YES];
}
@end

File: 第五章 产品化iOS应用（上）/连接社交网络/MZAccounts/ViewController.swift
//
//  ViewController.swift
//  MZAccounts
//
//  Created by 祝海焜 on 15/12/27.
//  Copyright © 2015年 Matin.Zhou. All rights reserved.
//

import UIKit
import Social

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func shareOnSinaWeibo(sender: AnyObject) {
        
        let shareOnSinaWeibo : SLComposeViewController = SLComposeViewController(forServiceType: SLServiceTypeSinaWeibo)
        
        self.presentViewController(shareOnSinaWeibo, animated: true, completion: nil)
        
    }


}

